src/containers/Guide/GuideBlocks/index.tsx

```diff
@@ -1,108 +1,118 @@
+        <A href="https://ed25519.cr.yp.to/">Ed25519 Digital Signature Algorithm</A>. This ensures that the account owner
+        has indeed signed all the transactions within a block.
       </p>
+      <DocCallout type={CalloutType.note}>
+        Although in many explanations of the network the account "owner" is often referred to by name, the actual
+        network architecture never stores the name of individuals. Instead, each account refers to the owner by their
+        account number.
+      </DocCallout>
       <p>
-        The account number (often referred to as the "public-key" in public-key cryptography) is not only used to
-        identify your account when other users wish to send you coins, but it is also used during the verification
-        process in which blocks must be correctly verified to ensure that the sender (account owner) has authorized the
-        related transactions. Therefore, a more realistic representation of a block would be as follows.
+        The account number (in public-key cryptography often referred to as the "public-key") is not only important for
+        identifying a user's account when others wish to send coins to this user. It is also important during the
+        verification process in which blocks must be correctly verified to ensure that the sender (account owner) has
+        authorized the related transactions. So a more realistic representation of a block is the following.
       </p>
-
       <DocImage alt="block details" maxWidth={530} src={BlockDetails} />
 
       <p>
-        Before we can go into more detail regarding exactly how a block is created, we first need to understand an
-        integral aspect of the network balance sheet called the balance lock. Every account on the balance sheet
-        includes a related balance lock. This is a value that the account owner must provide in order to "unlock" or
-        spend their coin balance. The method to unlock a balance lock is done through the use of balance keys, which are
-        provided within the message of every block.
+        Before discussing how a block is created, we must first focus on an integral aspect of the network balance sheet
+        called the <strong>balance lock</strong>.
       </p>
       <p>
-        Every message within a block will be hashed and that hash value will be used as the next balance lock. Since
-        every block contains a unique balance key (as created by the previous block), this will ensure that for every
-        block sent over the network the balance lock will become updated. Therefore, banks and validators will not
-        process the same block multiple times.
+        Every account on the balance sheet includes a related balance lock. This is a value that the account owner must
+        provide in order to "unlock" or spend their coin balance. The method to unlock a balance lock uses balance keys,
```

Comment:
I see we are using several double quotes. What do you think about using like italics or something along those lines for these?

Reply:
Generally, we can use ".." in more metaphoric concepts. I think this one is ok. I believe that italics are good, only if we define what they mean throught the tech doc set. We could easily get read of double quotes.